packages/server/middleware.js

```diff
@@ -66,6 +66,15 @@ module.exports.makeMiddleware = function({db, util}) {
       }
     ],
 
+    loadVarFromQueryOrBody: (key, required = true) => [
+      // Combines loadVarFromQuery and loadVarFromBody. The body recieves priority.
+
+      ...middleware.loadVarFromBody(key, false),
+      ...middleware.loadVarFromQuery(key, false), // Does not overwrite.
+
+      ...middleware.validateVar(key, required ? validate.defined : async x => true),
+    ],
+
```

Comment:
This is actually a super clever implementation, cool.

You should add tests to middleware.js, by the way (just take a look at the other two for example). Remember to test priority, i.e. that body is prioritized over query, as well as the "required" argument!